testAirBnbDirectory/airBnbCalendar-1.ics

```diff
@@ -0,0 +1,19 @@
+BEGIN:VCALENDAR
```

Comment:
Seems that the content of this directory should be part of .gitignore
And the directory should have better name

Reply:
Done!